### PR TITLE
✨ Share Button

### PR DESCRIPTION
**Pull request by DRSchlaubi**
https://twitter.com/SchlaumeierTVDE/status/1578759156772745216

### CODE REVIEW

**Comment by jheubuch** [comment]
I think that's a good idea, but I also think, that the "socialText"-Attribute of Träwelling-API could be used for sharing the status. Also by that we would lose the "one-click-check-in-clone"-feature... Do you have any ideas about moving the share button to another location? (like Floating Action Button whatever?)

**Comment by DRSchlaubi** [comment]
I thought about this for a while but I have no idea about where to put the share button

tbh i am not a huge fan of the floating action button

File: app/src/main/res/drawable/ic_location.xml [hunk 1]
```diff
@@ -1,10 +1,5 @@
-<vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="24dp"
-    android:height="32dp"
-    android:tint="?attr/colorControlNormal"
-    android:viewportWidth="384"
-    android:viewportHeight="512">
-    <path
-        android:fillColor="@android:color/white"
-        android:pathData="M215.7,499.2C267,435 384,279.4 384,192C384,86 298,0 192,0S0,86 0,192c0,87.4 117,243 168.3,307.2c12.3,15.3 35.1,15.3 47.4,0zM192,256c-35.3,0 -64,-28.7 -64,-64s28.7,-64 64,-64s64,28.7 64,64s-28.7,64 -64,64z" />
+<vector android:height="24dp" android:tint="#000000"
```
Comment (jheubuch) [inline app/src/main/res/drawable/ic_location.xml:1]:
```suggestion
<vector android:height="24dp" android:tint="?attr/colorControlNormal"
```

File: app/src/main/res/drawable/ic_home.xml [hunk 2]
```diff
@@ -1,10 +1,5 @@
-<vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="24dp"
-    android:height="24dp"
-    android:viewportWidth="24"
-    android:viewportHeight="24"
-    android:tint="?attr/colorControlNormal">
-  <path
-      android:fillColor="@android:color/darker_gray"
-      android:pathData="M10,20v-6h4v6h5v-8h3L12,3 2,12h3v8z"/>
+<vector android:height="24dp" android:tint="#000000"
```
Comment (jheubuch) [inline app/src/main/res/drawable/ic_home.xml:1]:
```suggestion
<vector android:height="24dp" android:tint="?attr/colorControlNormal"
```

**Review by jheubuch — CHANGES_REQUESTED** [review]

**Review by jheubuch — APPROVED** [review]

**Comment by jheubuch** [comment]
Please squash commits on merge and use Gitmoji for the commit message :)

**Pull request merged**